lua/tardis/sh_screens.lua

```diff
@@ -262,50 +276,180 @@ concommand.Add("tardis_toggleui", function()
 end)
 
 function TARDIS:LoadScreenUI(screen)
+	local background_img = TARDIS.visualgui_theme_basefolder
+	background_img = background_img..TARDIS:GetSetting("visual_gui_theme")
```

Comment:
From testing: The colour changing background still bleeds through the new UI, can we just render black behind this or you end up with a weird aliased glow around the screen
![image](https://user-images.githubusercontent.com/2363642/108837898-d443f880-75ca-11eb-9a2d-eecd439f6a0a.png)

Reply:
Closing this conversation with issue moved to: https://github.com/MattJeanes/TARDIS/issues/360#issuecomment-785286856